ta4j-examples/src/test/java/ta4jexamples/loaders/JsonBarsSerializerTest.java

```diff
@@ -23,18 +23,20 @@
  */
 package ta4jexamples.loaders;
 
-import org.junit.Rule;
-import org.junit.Test;
-import org.junit.rules.TemporaryFolder;
-import org.ta4j.core.Bar;
-import org.ta4j.core.BarSeries;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
 
 import java.io.File;
 import java.io.IOException;
 import java.math.BigDecimal;
 import java.util.concurrent.ThreadLocalRandom;
 
-import static org.junit.Assert.*;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
+import org.ta4j.core.Bar;
+import org.ta4j.core.BarSeries;
```

Comment:
@nimo23 you do a lot of imports swapping in different files that are not relevant in the specific PR

Reply:
Are you suggesting that I do such import swappings in a new PR in the future (I think such import swappings is ike formatting a file and does not deserve a PR).

Reply:
In future, I will release a PR when doing big reformats (or import swappings). Sorry for this inconvenience.